style.css

```diff
@@ -29,7 +29,7 @@ header {
 
 section {
   padding: 1rem;
-  height: calc(25vh - 2rem - 12.5px);
+  height: calc(20vh - 2rem - 10px);
```

Comment:
I feel like we could do something better here, but I might just make that a followup push…